modules/ROOT/pages/batch-error-handling-faq.adoc

```diff
@@ -1,23 +1,33 @@
-= Handling Errors During Batch Job
+= Handling Errors During Batch Jobs
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
 == Overview
 
-Handling errors inside a batch job can be complicated considering the huge amount of data this scope is designed to take. +
-When processing 1 million records, it's almost impossible to log everything. Logs would become huge, unreadable, and would take a massive toll on the performance of your processing. +
-On the other hand, logging too little would make it impossible for you to know what went wrong. And considering that if out of those 1 million records, 30 thousand of them failed, not knowing what’s wrong with them can turn your debugging task very problematic. This trade-off is not simple to overcome.
+Mule batch processing is designed to handle very large data sets and to perform "near real-time" data integration that recovers from crashes and continues processing a job from a point of failure. However, _verbose_ logs for issues that occur in large data sets can become enormous and severely impact performance. 
 
+To limit this impact, Mule uses INFO-level logging by default, as described in <<default_logging>>. For cases in which you require more verbose log messages, you can change the mode to DEBUG. This mode is helpful for debugging and is feasible for some cases that involve smaller data sets. 
+
+The following property is available for setting the logging mode:
```

Comment:
Why this general specification about logging is needed in batch?

Reply:
@fsgonz I believe it was added (to the original) because the INFO mode summaries do not capture the info about records that someone might want to see. See in the steps (unedited): "Even if all records raise the same error, the messages being processed would probably contain specific information related to those records. For example, if I'm pushing leads to my Salesforce account and one record fails because the lead was already uploaded, another repeated lead would have different record information, but the error is the same."

Reply:
Also, there are steps on this page that describe the default process in detail, I believe because the default logging behavior might be unexpected.